### ISSUE TITLE
PyCBC virtualenv instructions need to include GraceDB modules

### ISSUE THREAD

**Issue by spxiwh** [ligo-cbc/pycbc#431]
The pycbc script pycbc_upload_xml_to_gracedb requires the gracedb libraries to upload triggers. Currently these libraries are not present if one follows the virtualenv install instructions:

(pycbc_er8b_production)[spxiwh@atlas7 09:44 AM ~]$ pycbc_upload_xml_to_gracedb --help
Traceback (most recent call last):
  File "/home/tito/sw_install/pycbc_er8b_production/bin/pycbc_upload_xml_to_gracedb", line 4, in <module>
    __import__('pkg_resources').run_script('PyCBC===0ad758', 'pycbc_upload_xml_to_gracedb')
  File "/home/tito/sw_install/pycbc_er8b_production/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 735, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/home/tito/sw_install/pycbc_er8b_production/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 1652, in run_script
    exec(code, namespace, namespace)
  File "/home/tito/sw_install/pycbc_er8b_production/lib/python2.7/site-packages/PyCBC-0ad758-py2.7.egg/EGG-INFO/scripts/pycbc_upload_xml_to_gracedb", line 26, in <module>
    from ligo.gracedb.rest import GraceDb
ImportError: No module named ligo.gracedb.rest

**Comment by duncan-brown** [comment]
Does 
```shell
pip install ligo-gracedb
```
fix the problem? If so, please submit a patch to the instructions.

**Comment by spxiwh** [comment]
Yes, this does fix the problem. I didn't realise/expect this would be on pip!

BTW, the system install of gracedb on SYR is *very* old.

**Comment by spxiwh** [comment]
I will prepare a patch

**Comment by duncan-brown** [comment]
```shell
[root@sugar-dev3 ~]# rpm -e ligo-gracedb
```

**Comment by spxiwh** [comment]
Patch is coming, but before closing this issue, can I ask @lppekows to verify that the prepacked binary for pycbc_upload_xml_to_gracedb is linked against the correct library of gracedb?

**Comment by lppekows** [comment]
Hi Ian,

No, it currently doesn't.  Without special handling the packaged binaries will pick up whatever's in my environment at build time, so 
```
$ pycbc_upload_xml_to_gracedb
ImportError: No module named ligo.gracedb.rest
```

In fact it's worse than that, since this program has some unhandled hidden imports
```
$ ~/Repos/pycbc-software/v1.2.0/x86_64/composer_xe_2015.0.090/pycbc_upload_xml_to_gracedb 
ImportError: No module named cython_blas
```

I've added an issue to address this for 1.2.2.

**Comment by spxiwh** [comment]
Fixed with #438 and #439.
